server/src/main/java/com/linecorp/centraldogma/server/internal/storage/repository/git/GitRepository.java

```diff
@@ -136,6 +139,18 @@ class GitRepository implements Repository {
     private static final Field revWalkObjectsField;
 
     static {
+        final String jgitPomProperties = "META-INF/maven/org.eclipse.jgit/org.eclipse.jgit/pom.properties";
+        try (InputStream is = SystemReader.class.getClassLoader().getResourceAsStream(jgitPomProperties)) {
+            final Properties props = new Properties();
+            props.load(is);
+            final Object jgitVersion = props.get("version");
+            if (jgitVersion != null) {
+                logger.info("Using JGit: {}", jgitVersion);
+            }
+        } catch (IOException e) {
+            logger.debug("Failed to read JGit version", e);
```

Comment:
warn would be better?

Reply:
Debug level used in case it is intentionally removed while shadowing tasksby users.